govtool/frontend/src/hooks/useWalletErrorModal.tsx

```diff
@@ -0,0 +1,43 @@
+import { useModal } from "@context";
+import { useTranslation } from "react-i18next";
+
+type WalletErrorModalProps = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  error: any;
```

Comment:
Can it be just unknown?

Reply:
I've set `unknown` and added:
`const errorMessage = (error && typeof error === 'object' && 'info' in error) ? error.info : error;`